src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/EnumDataTypeAttribute.cs

```diff
@@ -76,14 +76,9 @@ public override bool IsValid(object value)
             {
                 try
                 {
-                    if (stringValue != null)
-                    {
-                        convertedValue = Enum.Parse(EnumType, stringValue, false);
-                    }
-                    else
-                    {
-                        convertedValue = Enum.ToObject(EnumType, value);
-                    }
+                    convertedValue = stringValue != null
+                        ? Enum.Parse(EnumType, stringValue, false)
+                        : Enum.ToObject(EnumType, value);
```

Comment:
I don't see advantage of this change

Reply:
I prefer conditional operator, so good with this.